src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java

```diff
@@ -246,19 +258,23 @@ private ItemRepository(Clients clients, LocationRepository locationRepository,
         .distinct()
         .collect(Collectors.toList());
 
-      final FindWithMultipleCqlIndexValues<JsonObject> fetcher
-        = findWithMultipleCqlIndexValues(instancesClient, "instances", identity());
-
       final var mapper = new InstanceMapper();
 
-      return fetcher.findByIds(instanceIds)
+      return fetchInstancesByIds(instanceIds)
         .thenApply(r -> r.map(instances -> items.stream()
           .map(item -> item.withInstance(mapper.toDomain(
               findById(item.getInstanceId(), instances.getRecords()).orElse(null))))
           .collect(Collectors.toList())));
     });
   }
 
+  private CompletableFuture<Result<MultipleRecords<JsonObject>>> fetchInstancesByIds(
+    Collection<String> ids) {
+
+    return findWithMultipleCqlIndexValues(instancesClient, INSTANCES_RECORDS_PROPERTY_NAME,
+      identity()).findByIds(ids);
```

Comment:
Given that `INSTANCES_RECORDS_PROPERTY_NAME` is used only once in this private method, could this be a constant in the method (or even inline) in order to keep it within the same scope as it's usage (and make refactoring easier)?

Reply:
fixed, I just updated method by merging and forgot to make it inline, thanks.

Reply:
Thank you. 

Can a similar refactoring be done for the `HOLDINGS_RECORDS_COLLECTION_PROPERTY_NAME`?

Reply:
It is used more than once actually and seems not good idea to make inline on multiple places

Reply:
Agreed. 

My interpretation of other changes in this pull request was that the `fetchInstancesByID` method was extracted in order to reduce similar duplication for the instance fetcher. 

I wondered if the same could be done for the holdings fetcher.

If not, please resolve this conversation.

Reply:
ok, thanks

Reply:
ok, extracted duplication part, and also made `HOLDINGS_RECORDS_PROPERTY_NAME` inline